protobuf-solidity/src/protoc/plugin/gen_sol.py

```diff
@@ -300,22 +308,29 @@ def generate_code(request, response):
       continue
     # main output
     output = []
+    output_path = gen_output_path(proto_file)
 
     # generate sol library
     # prologue
     output.append('// SPDX-License-Identifier: Apache-2.0\npragma solidity ^{0};'.format(util.SOLIDITY_VERSION))
     for pragma in util.SOLIDITY_PRAGMAS:
       output.append('{0};'.format(pragma))
-    output.append('import "{0}";'.format(RUNTIME_FILE_NAME))
-    output.append('import "{0}";'.format(PROTOBUF_ANY_FILE_NAME))
+    global GEN_RUNTIME
```

Comment:
@siburu this `global` isn't required here due to read-only in the scope.

Reply:
@bluele I see. Python is difficult...
I'll remove this. Wait for a minute.

Reply:
@bluele done!